src/main/webapp/index.html

```diff
@@ -35,18 +34,21 @@ <h4 class="search-title">Filter the Graph</h4>
     <!-- Box for zooming in -->
     <div id="zoom-box" class="input-child">
       <h4 class="search-title">Zoom In</h4>
-        <label>Zoom on Node in Graph: </label>
-        <input type="text" id="node-search" placeholder="Node Name"></input>
-        <button id="search-button">Find Node</button>
-        <label id="node-error" class="error-text"></label>
+      <label>Zoom on Node in Graph: </label>
+      <input type="text" id="node-search" placeholder="Node Name"></input>
+      <button id="search-node-button">Find Node</button>
 
-        <label>Zoom on Token in Graph: </label>
-        <input type="text" id="token-search" placeholder="Token Name"></input>
-        <button id="search-token-button">Find Token</button>
-        <label id="token-error" class="error-text"></label>
-  
-      <button id="reset" class="reset">Reset Zoom Effects</button>
-      <label id="num-selected"></label>
+      <label>Zoom on Token in Graph: </label>
+      <input type="text" id="token-search" placeholder="Token Name"></input>
+      <button id="search-token-button">Find Token</button>
+      <div>
+        <label for="highlight-number">Highlighted Token: </label>
+        <input type="number" id="highlight-number" min="0" value="0" max="0" disabled=true>
+        <label id="num-selected">out of 0</label>
+        <button id="prevnode" class="material-icons" disabled=true>navigate_before</button>
```

Comment:
consider hiding this when no token is searched & when there's one searched? it can be a bit confusing for the user

Reply:
atm they are disabled in those scenarios, do you think not displaying them at all would be better?